ui/packages/consul-ui/app/adapters/node.js

```diff
@@ -11,12 +11,12 @@ import { inject as service } from '@ember/service';
 // to the node.
 
 export default class NodeAdapter extends Adapter {
-  @service features;
+  @service abilities;
 
   get peeringQuery() {
     const query = {};
 
-    if (this.features.isEnabled('peering')) {
+    if (this.abilities.can('use peers')) {
```

Comment:
I'm not totally sure if we need this now, I think originally we only had this guard here as we wanted to merge this and then release without this work being enabled. I think moving forwards we always need to send `with-peers` whether the cluster you are on has peering enabled or not. But! Flagging peering has changed quite a bit, so I'm not sure if this is still the case, I'll check in with folks here and see.